### PR TITLE
Quick fix for failing account linking

### PR DESCRIPTION
**Pull request by smashah**
Code can't find client id and secret. Since it's predetermined just hardcode it so you can move on with your life and continue learning.

### CODE REVIEW

**Comment by smashah** [comment]
I signed it!

**Comment by Fleker** [comment]
I do appreciate the change, though I am a bit curious why the code will sometimes fail. Fixing that would be more useful, although if we can't figure it out then let's merge in this patch.

File: smart-home-provider/cloud/auth-provider.js [hunk 1]
```diff
@@ -26,6 +26,9 @@ const util = require('util');
 // eslint-disable-next-line no-unused-vars
 const session = require('express-session');
 
+const clientId = "ENTER CLIENT ID HERE"
```
Comment (Fleker) [inline smart-home-provider/cloud/auth-provider.js]: Shouldn't these come from our `config-provider.js`?

**Comment by GermanBluefox** [comment]
I cannot understand these changes too.
It will accept any clientId/secretId without check.

**Pull request closed without merging**